src/components/date-range/date-range.stories.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import notes from './notes.md';
+import DateRange from './date-range.js';
+
+const store = new Store({
+  value: ['2016-10-01', '2016-10-30']
+});
+const handleChange = (newDate) => {
+  store.set({ value: newDate });
+  action('changed')(newDate);
+};
+
+storiesOf('DateRange', module)
```

Comment:
I know that this also happens in the current demo site, but ideally I wouldn't want the dropdown to cover the information.
![image](https://user-images.githubusercontent.com/26925206/53880535-228c1580-4009-11e9-9299-b6bdc15e9587.png)

Reply:
![Zrzut ekranu 2019-03-20 o 10 10 37](https://user-images.githubusercontent.com/12142840/54672500-7f52fa00-4af8-11e9-8106-a247095d4256.png)

Reply:
Currently the story source is uncovered. Is it OK?